roles/satellite-clone/defaults/main.yml

```diff
@@ -17,3 +17,5 @@ skip_satellite_rpm_check: False
 overwrite_etc_hosts: True
 puppet_version: 3
 check_networking_interfaces: True
+sp_features_table_name: smart_proxy_features
+satellite_service: katello-service
```

Comment:
I would rather default to what it is now `satellite-maintain service` rather than what the previous versions used. 

6.7 and earlier vars files need to be updated, but we won't have to keep adding this param